WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -79,9 +79,9 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "FailFastOutputRegistrationTimeout", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public TimeSpan FailFastOutputRegistrationTimeout { get; set; } = TimeSpan.FromMinutes(3);
 
-	[DefaultValueTimeSpan("0d 0h 1m 0s")]
+	[DefaultValueTimeSpan("0d 0h 0m 30s")]
 	[JsonProperty(PropertyName = "FailFastTransactionSigningTimeout", DefaultValueHandling = DefaultValueHandling.Populate)]
-	public TimeSpan FailFastTransactionSigningTimeout { get; set; } = TimeSpan.FromMinutes(1);
+	public TimeSpan FailFastTransactionSigningTimeout { get; set; } = TimeSpan.FromMinutes(0.5);
```

Comment:
I'm not sure if we should reduce the default `TimeSpan` for this, since you guys asked us to increase this property in the `WabiSabiConfig` to `1m30s`.

Reply:
The point is that users who don't sign the transaction won't be banned. Which means it is ok for trezor users not to sign the transaction because it has not negative consequences.

Reply:
What is the reason to change the default? 

After this PR we will change conf in prod manually - that is the only way even if the default is changed. It will only use the default when the config file is newly created.